rollup/configs/babel-presets.js

```diff
@@ -1,9 +1,8 @@
 export const packageBabelPreset = [
 	['@babel/preset-env', {
-		corejs: 3,
```

Comment:
I had already fixed this in the `v8` branch.
We don't have `corejs` as a dependency.
This worked because we only wrote JS that didn't need any polyfills from `corejs`.

`css-tree` might require a polyfill for typed arrays. (these matches can have false positives)

Removing `corejs` from our config is the right fix as we never actually had it in the first place.

Reply:
Agreed!